.gitignore

```diff
@@ -1,2 +1,3 @@
+/Cargo.lock
```

Comment:
So, my thinking to date has been that for most people, the main deliverable is the `tectonic` command line program, not the crate, so that keeping the `Cargo.lock` in version control is the way to go.

I'm not sure the best approach. The reusable crate could just be separated out into a sub-project with its own `Cargo.toml`, of course. But I got the impression that it's not an uncommon pattern for Rust tools to have a single `Cargo.toml` for a featureful crate plus a command-line interface to it ... I should do some digging to figure out the best practices, or ask on [the Rust forums](https://users.rust-lang.org/).